apps/src/code-studio/pd/workshop_dashboard/components/date_picker.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * DatePicker control.
+ * It's basically a wrapper around react-datepicker (with limited props) that displays
+ * as a React-Bootstrap select with a calendar icon Addon.
+ */
```

Comment:
It might not be worth your time to do this now, but the next time you add something like this it might be helpful to add a storybook entry for it. It can save some time since you don't have to wait for your javascript to rebuild, plus then you have a convenient demo for anyone that wants to reuse the component. see https://github.com/code-dot-org/code-dot-org/blob/staging/apps/README.md#ui-component-style-guide